### ISSUE TITLE
Unable to run pod install

### ISSUE THREAD

**Issue by julienvincent** [tlenclos/react-native-audio-streaming#17]
I get a cocoa error when running pod install:

```
untimeError - [Xcodeproj] Consistency issue: no parent for object `Main Group`: ``
```

**Comment by madhu314** [comment]
Ran into similar issue, seems to be issue with cocoapods, https://github.com/CocoaPods/CocoaPods/issues/5847#issuecomment-248617413 suggestion helped me resolve tit

**Comment by efhayati** [comment]
@madhu314 Would you please let me know how did you resolve it? I tries the suggested URL but I couldn't fix the issue.

**Comment by julienvincent** [comment]
I 'solved' it by copy pasting the two source (`ReactNativeAudioStreaming.h` and ReactNativeAudioStreaming.m`) files from github directly into my project.

**Comment by madhu314** [comment]
I think latest master should have the fix, issue (https://github.com/tlenclos/react-native-audio-streaming/issues/10)[#10] has been closed with a PR

**Comment by tlenclos** [comment]
This should be fixed with latest release (2.2.0), can you retry please ?
